integration/aspnet_test.go

```diff
@@ -1,129 +0,0 @@
-package integration_test
-
-import (
-	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
-	"path/filepath"
-	"testing"
-
-	"github.com/paketo-buildpacks/occam"
-	"github.com/sclevine/spec"
-
-	. "github.com/onsi/gomega"
-	. "github.com/paketo-buildpacks/occam/matchers"
-)
-
-func testAspnetSource(t *testing.T, context spec.G, it spec.S) {
-	var (
-		Expect     = NewWithT(t).Expect
-		Eventually = NewWithT(t).Eventually
-		pack       occam.Pack
-		docker     occam.Docker
-	)
-
-	it.Before(func() {
-		pack = occam.NewPack()
-		docker = occam.NewDocker()
-	})
-
-	context("when building an app", func() {
-		var (
-			image     occam.Image
-			container occam.Container
-			name      string
-			source    string
-		)
-
-		it.Before(func() {
-			var err error
-			name, err = occam.RandomName()
-			Expect(err).NotTo(HaveOccurred())
-		})
-
-		it.After(func() {
-			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
-			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
-			Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
-			Expect(os.RemoveAll(source)).To(Succeed())
-		})
-
-		it("should build a working OCI image for a source_aspnetcore_all_2.1 application", func() {
```

Comment:
Seems like these tests are supposed to make sure that the dotnet-publish buildpack correctly `requires` ASPNET when it finds certain package references ([AspNetCore.App](https://github.com/paketo-buildpacks/dotnet-publish/blob/c49c431209299a84536713b40a2ac1bb0b6d8708/integration/testdata/source_aspnetcore_app_2.1/source_aspnetcore_2.1.csproj#L12) and [AspNetCore.All](https://github.com/paketo-buildpacks/dotnet-publish/blob/c49c431209299a84536713b40a2ac1bb0b6d8708/integration/testdata/source_aspnetcore_all_2.1/netcoreapp2.csproj#L8)) in the projfile. However this detection logic is [covered in the project file parser tests](https://github.com/paketo-buildpacks/dotnet-publish/blob/main/dotnet_project_file_parser_test.go#L117-L152) so I don't think we need an integration test.